setuptools_rust/extension.py

```diff
@@ -113,6 +114,7 @@ def __init__(
         rustc_flags: Optional[List[str]] = None,
         rust_version: Optional[str] = None,
         quiet: bool = False,
+        locked: bool = False,
```

Comment:
Locked is fine, though as noted should this instead be a more generic `cargo_manifest_args: List[str] = []`?

Reply:
Updated to make generic over `locked`, `frozen` and `offline`